src/oss-find-squats-lib/FindPackageSquatResult.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.CST.OpenSource.FindSquats.Mutators;
```

Comment:
Looks like the copyright header is missing in a couple of new files (I _think_ with the new linting rules you have in place, you can enforce a copyright header)

Reply:
Found a way to do with editorconfig. Thanks!